--- krellix-0.1.2-orig/src/krellix/SettingsDialog.cpp	2026-05-31 19:00:37.185350726 -0400
+++ krellix-0.1.2/src/krellix/SettingsDialog.cpp	2026-05-31 19:00:50.507280449 -0400
@@ -86,6 +86,7 @@
     {QStringLiteral("host"),    QStringLiteral("Host")},
     {QStringLiteral("cpu"),     QStringLiteral("CPU")},
     {QStringLiteral("proc"),    QStringLiteral("Proc")},
+    {QStringLiteral("krelltop"), QStringLiteral("KrellTop")},
     {QStringLiteral("mem"),     QStringLiteral("Memory + Swap")},
     {QStringLiteral("uptime"),  QStringLiteral("Uptime")},
     {QStringLiteral("net"),     QStringLiteral("Net")},
